packages/prisma-client/.gitignore

```diff
@@ -0,0 +1 @@
+src/_generated
```

Comment:
We use `__generated__` everywhere 8n the project

Reply:
roger that, I adjusted the name of this directory